demo/index.html

```diff
@@ -16,12 +16,68 @@
 
     <demo-navigation></demo-navigation>
 
-    <h3>Sample example</h3>
+    <h3>Default Radio</h3>
     <demo-snippet>
       <template>
-        <vaadin-radio-button>Foobar</vaadin-radio-button>
+        <vaadin-radio-button>Radio button</vaadin-radio-button>
       </template>
     </demo-snippet>
+
+    <h3>Radio Group</h3>
+    <demo-snippet>
+      <template>
+        <vaadin-radio-group name="radio-group" value="fourth">
+          <vaadin-radio-button><b>1</b></vaadin-radio-button>
```

Comment:
Moved to iron form section in https://github.com/vaadin/vaadin-radio-button/pull/2